.rubocop.yml

```diff
@@ -15,7 +15,7 @@ Layout/IndentationConsistency:
 Metrics/MethodLength:
   Exclude:
     - "app/controllers/families_controller.rb"
```

Comment:
This line should be removed. Maybe instead, increase the max lines to about 11?

Reply:
👌